src/Core/Components/DataGrid/FluentDataGrid.razor

```diff
@@ -29,19 +29,34 @@
                     @_renderColumnHeaders
                 </FluentDataGridRow>
             }
-
-            @if (Virtualize)
+            @if (Items?.Count() == 0 || _ariaBodyRowCount == 0)
```

Comment:
It's better to use `.Any()`... to avoid to compute the number of all items.

Reply:
True, but in this case using `.Any()` gives Operator '||' cannot be applied to operands of type 'bool?' and 'bool'. Choose the easy way out :)